train_llama3.cu

```diff
@@ -1004,11 +1059,12 @@ float gpt2_calculate_grad_norm(GPT2 *model, MultiGpuConfig* multi_gpu_config) {
         // grads_memory only contains the averaged gradients at the local shards,
         // so we only calculate the grad norm at the grads_memory belonging to the local shards
         for (int i = 0; i < NUM_PARAMETER_TENSORS; i++) {
+            if (!gated_ffn && (i == 12 || i == 13)) { continue; }  // skip the gated ffn weights
```

Comment:
is this just for the case when the activation function is not SwiGLU, so there are no gated ffn weights? If so the comment is a bit misleading